File: qb-engineer-ui/src/app/features/sales-orders/models/sales-order-return.model.ts
export interface SalesOrderReturn {
  id: number;
  returnNumber: string;
  status: string;
  reason: string | null;
  returnDate: Date | null;
  originalJobId: number | null;
  originalJobNumber: string | null;
  reworkJobId: number | null;
  reworkJobNumber: string | null;
  inspectionNotes: string | null;
}

File: qb-engineer-server/qb-engineer.core/Models/SalesOrderReturnModel.cs
namespace QBEngineer.Core.Models;

public record SalesOrderReturnModel(
    int Id,
    string ReturnNumber,
    string Status,
    string? Reason,
    DateTimeOffset? ReturnDate,
    int? OriginalJobId,
    string? OriginalJobNumber,
    int? ReworkJobId,
    string? ReworkJobNumber,
    string? InspectionNotes);
